enhancements/ingress/set-delete-http-headers.md

```diff
@@ -1001,17 +1000,45 @@ This route can be annotated with `haproxy.router.openshift.io/disable_cookies=tr
   must not exceed the value of spec.tuningOptions.headerBufferMaxRewriteBytes on the
   IngressController.
 
-#### Why does a custom header value defined in RouteSpec override IngressControllerSpec value ?
-  Answer: It's quite common to have a global setting with an escape hatch locally to override the global setting.
-  The purpose of this EP is to define custom headers. So, there are two reasons we need such override:
-  1. A route owner shall have the choice to define what headers he wants as per his application requirements(as they might be varying) 
-  than the ones admin has set cluster-wide.
-  2. The IngressController and Route might be owned by the same person who wants to specify some global configuration
-  and then override that global configuration on a small subset of routes. 
+#### How do Route headers interact with IngressController headers of the same value?
+  Answer:  It depends on if they are response headers or request headers.
+  
+ In case of HTTP response headers, the headers set in spec.httpHeaders.actions of IngressController will override
+ the headers with same name in Route's spec.httpHeaders.actions or Route annotation.
```

Comment:
It is more precise to describe the order in which actions are executed.  Also, change "In case" to "In the case".  
```suggestion
 In case of HTTP response headers, the actions specified in `spec.httpHeaders.actions` on the IngressController will be executed after
 the actions specified in the Route's `spec.httpHeaders.actions` field.
```
Please remember to use backticks when referencing API fields.

Reply:
I agree - please describe the order in which the actions are executed.

Reply:
yes

Reply:
done